app/services/food_service.rb

```diff
@@ -0,0 +1,11 @@
+class FoodService
+  def self.search(search_term)
+    query = search_term.gsub(' ','%20') 
```

Comment:
I believe Faraday should be smart enough for this substitution to not be necessary (i.e. you should be able to pass in a query with spaces